src/test/kotlin/no/digdir/informasjonsforvaltning/fdk_dataservice_harvester/contract/DataServicesContract.kt

```diff
@@ -0,0 +1,46 @@
+package no.digdir.informasjonsforvaltning.fdk_dataservice_harvester.contract
+
+import no.digdir.informasjonsforvaltning.fdk_dataservice_harvester.utils.ApiTestContainer
+import no.digdir.informasjonsforvaltning.fdk_dataservice_harvester.utils.DATASERVICE_ID_0
+import no.digdir.informasjonsforvaltning.fdk_dataservice_harvester.utils.TestResponseReader
+import no.digdir.informasjonsforvaltning.fdk_dataservice_harvester.utils.apiGet
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Tag
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.springframework.http.HttpStatus
+import kotlin.test.assertTrue
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@Tag("contract")
+class DataServicesContract : ApiTestContainer() {
+    private val responseReader = TestResponseReader()
+
+    @Test
+    fun findSpecific() {
+        val response = apiGet("/dataservices/$DATASERVICE_ID_0", "application/rdf+json")
+        assertEquals(HttpStatus.OK.value(), response["status"])
```

Comment:
https://junit.org/junit5/docs/5.0.0/api/org/junit/jupiter/api/Assumptions.html

Reply:
Om dette er et utgangspunkt for at resten av testen skal gjennomføres gir det mer mening å bruke Assumptions enn Assertions

Reply:
58621dc

Sånn?